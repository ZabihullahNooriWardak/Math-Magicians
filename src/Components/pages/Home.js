export default function Home() {
  return (
    <div className="home">
      <h1>Welcome to MathMagicians</h1>
      <p>
        Step into the captivating realm of Math Magician, an exceptional
        digital space that seamlessly marries the art of calculation with
        the inspiration of language.

      </p>
      <p>
        As you embark on this digital journey, you will find yourself equipped
        with an innovative calculator that stands ready to unravel the
        complexities of numbers with precision and efficiency. Whether
        you are navigating everyday computations or delving into more
        intricate mathematical puzzles, our calculator offers a user-friendly
        interface that makes math not just a task, but a delightful exploration.

      </p>
      <p>
        But Math Magician is more than just a calculator; it is a haven for those
        who seek intellectual enrichment and a touch of mindfulness.
        Venture beyond the realm of numbers, and you will discover a treasure trove
        of wisdom through our carefully curated selection of random quotes.
        These quotes, sourced from diverse minds and backgrounds, provide moments
        of introspection and inspiration that beautifully complement your mathematical
        endeavors.

      </p>
      <p>
        Our commitment goes beyond functionality  it is about fostering a holistic
        experience. We understand that math is not just about equations; it is
        a journey of discovery, problem-solving, and growth.
        And as you immerse yourself in the wonders of Math Magician,
        you are not just crunching numbers; you are nurturing your curiosity,
        refining your skills, and nurturing your appreciation for the artistry
        that exists within both mathematics and language.

      </p>
      <p>
        Whether you are a student seeking a tool to assist with studies, a professional
        looking for quick and accurate calculations, or simply an individual with
        an appreciation for the finer aspects of life, Math Magician invites you to
        explore, engage, and elevate. Join us in celebrating the synergy of logic and
        expression, calculation and contemplation. Together, let us unlock the true magic
        that lies at the intersection of mathematics and the human spirit.

      </p>
    </div>
  );
}
