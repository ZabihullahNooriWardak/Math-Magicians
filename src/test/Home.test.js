import React from 'react';
import { render } from '@testing-library/react';
import Home from '../Components/pages/Home';

test('render correctly', () => {
  const homeComponent = render(<Home />);
  expect(homeComponent).toMatchSnapshot();
});
