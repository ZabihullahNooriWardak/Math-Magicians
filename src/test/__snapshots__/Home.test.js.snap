// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render correctly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="home"
      >
        <h1>
          Welcome to MathMagicians
        </h1>
        <p>
          Step into the captivating realm of Math Magician, an exceptional digital space that seamlessly marries the art of calculation with the inspiration of language.
        </p>
        <p>
          As you embark on this digital journey, you will find yourself equipped with an innovative calculator that stands ready to unravel the complexities of numbers with precision and efficiency. Whether you are navigating everyday computations or delving into more intricate mathematical puzzles, our calculator offers a user-friendly interface that makes math not just a task, but a delightful exploration.
        </p>
        <p>
          But Math Magician is more than just a calculator; it is a haven for those who seek intellectual enrichment and a touch of mindfulness. Venture beyond the realm of numbers, and you will discover a treasure trove of wisdom through our carefully curated selection of random quotes. These quotes, sourced from diverse minds and backgrounds, provide moments of introspection and inspiration that beautifully complement your mathematical endeavors.
        </p>
        <p>
          Our commitment goes beyond functionality  it is about fostering a holistic experience. We understand that math is not just about equations; it is a journey of discovery, problem-solving, and growth. And as you immerse yourself in the wonders of Math Magician, you are not just crunching numbers; you are nurturing your curiosity, refining your skills, and nurturing your appreciation for the artistry that exists within both mathematics and language.
        </p>
        <p>
          Whether you are a student seeking a tool to assist with studies, a professional looking for quick and accurate calculations, or simply an individual with an appreciation for the finer aspects of life, Math Magician invites you to explore, engage, and elevate. Join us in celebrating the synergy of logic and expression, calculation and contemplation. Together, let us unlock the true magic that lies at the intersection of mathematics and the human spirit.
        </p>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="home"
    >
      <h1>
        Welcome to MathMagicians
      </h1>
      <p>
        Step into the captivating realm of Math Magician, an exceptional digital space that seamlessly marries the art of calculation with the inspiration of language.
      </p>
      <p>
        As you embark on this digital journey, you will find yourself equipped with an innovative calculator that stands ready to unravel the complexities of numbers with precision and efficiency. Whether you are navigating everyday computations or delving into more intricate mathematical puzzles, our calculator offers a user-friendly interface that makes math not just a task, but a delightful exploration.
      </p>
      <p>
        But Math Magician is more than just a calculator; it is a haven for those who seek intellectual enrichment and a touch of mindfulness. Venture beyond the realm of numbers, and you will discover a treasure trove of wisdom through our carefully curated selection of random quotes. These quotes, sourced from diverse minds and backgrounds, provide moments of introspection and inspiration that beautifully complement your mathematical endeavors.
      </p>
      <p>
        Our commitment goes beyond functionality  it is about fostering a holistic experience. We understand that math is not just about equations; it is a journey of discovery, problem-solving, and growth. And as you immerse yourself in the wonders of Math Magician, you are not just crunching numbers; you are nurturing your curiosity, refining your skills, and nurturing your appreciation for the artistry that exists within both mathematics and language.
      </p>
      <p>
        Whether you are a student seeking a tool to assist with studies, a professional looking for quick and accurate calculations, or simply an individual with an appreciation for the finer aspects of life, Math Magician invites you to explore, engage, and elevate. Join us in celebrating the synergy of logic and expression, calculation and contemplation. Together, let us unlock the true magic that lies at the intersection of mathematics and the human spirit.
      </p>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
